src/Form/AbstractForm.php

```diff
@@ -634,8 +634,8 @@ public static function getInputClassName($inputType)
             'checkbox' => CheckboxField::class,
             'radio' => RadioField::class,
             'select' => SelectField::class,
-            'media' => rex_form_widget_media_element::class,
-            'link' => rex_form_widget_linkmap_element::class,
+            'media' => MediaField::class,
+            'link' => ArticleField::class,
```

Comment:
Weiß gerade aus dem Kopf nicht, wo dieses `link` herkommt. Aber eventuell sollte/muss es auch in `article` geändert werden? Aber eventuell dann auch noch weitere Stellen? Oder die Stellen besser erstmal bei `link` lassen?